specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -7557,6 +7569,317 @@
           }
         }
       }
+    },
+    "/apps/{appId}/azureaccounts": {
+      "post": {
+        "description": "Assigns an azure account to the application.",
+        "operationId": "5be32228e8473de116325515",
+        "summary": "apps - Assign a LUIS azure account to an application",
+        "parameters": [
+          {
+            "$ref": "#/parameters/AppIdInPath"
+          },
+          {
+            "$ref": "#/parameters/BearerAuthPath"
+          },
+          {
+            "name": "azureAccountInfoObject",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/AzureAccountInfoObject"
+            },
+            "description": "The azure account information object."
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/OperationStatus"
+            }
+          },
+          "429": {
```

Comment:
Error responses are not supported by autorest as of now.

Add the error response to the default section. This applies all through this file.

Reply:
@praries880 Can you explain more as the SDKs were generated successfully and the error responses were included in the SDKs with no problem like [here](https://github.com/Azure/azure-sdk-for-js/pull/574/files#diff-ea23d7be7ea8160bc42df89b5fcb558fR258).

Reply:
Any update on this? @praries880

Reply:
@praries880 any updates ? this is blocking our release before the holidays, appreciate if we can close on that before end of week